src/app/hearing/components/create-hearing-note/create-hearing-note.component.html

```diff
@@ -0,0 +1,6 @@
+<div fxLayout="row" fxLayoutAlign="start start">
```

Comment:
@r-kruk-kainos What I know this **fxLayoutAlign="start start"** was causing UI issue on IE. @andrewtaylorkainos plz correct me if I'm wrong.

@r-kruk-kainos  can you make sure that its display ok on IE?

Reply:
I've checked it on IE and it was displaying ok.